types/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export = get;
+declare function get(src: any, type: string): Promise<any>;
```

Comment:
The second property type is actually optional. Please check the implementation here.  https://github.com/kabirbaidhya/get-js/blob/master/src/get.js#L89

You'll need this. 
```suggestion
declare function get(src: any, type?: string): Promise<any>;
```

Reply:
Never mind, it's a simple change - I'll do it.